docs/application/native/guides/machine-learning/overview.md

```diff
@@ -0,0 +1,14 @@
+# Machine-Learining
+
+
+Machine-learning features introduce how you can handle machine learning frameworks, like tensorflow, tensorflow-lite, with the construction of data pipeline.
+
+You can use the following machine-learning features in your native applications:
+
+- [Machine Learning Inference C-API](machine-learning-inference.md)
+
+  You can construct the data pipeline which contains the machine learning model filter. The pipeline can be assembled and controlled with various elements of [gstreamer](https://gstreamer.freedesktop.org/). 
+
+## Related Information
+- Dependencies
+  - Tizen 5.5 and Higher
```

Comment:
As this page is Native guide, please follow the form used for Native guides.

  - Tizen 5.5 and Higher for Mobile
  - Tizen 5.5 and Higher for Wearable